test/functional/sqlsrv/0020.phpt

```diff
@@ -33,74 +32,83 @@ function RunTest( $field_type ) {
         
     while (($originalLine = fread($originalStream, 80)) &&
             ($dbLine = fread($stream, 80))) {
-        if( $originalLine != $dbLine )
-            die( "Not identical" );
+        if ($originalLine != $dbLine) {
+            die("Not identical");
+        }
     }
 
+    dropTable($conn, $params['tableName']);
+    
     sqlsrv_free_stmt($stmt) || die(print_r(sqlsrv_errors(), true));
 
     sqlsrv_close($conn) || die(print_r(sqlsrv_errors(), true));
+    
 }
 
-RunTest( "varchar(max)" );
-RunTest( "varbinary(max)" );
-RunTest( "nvarchar(max)" );
+runTest("varchar(max)");
+runTest("varbinary(max)");
+runTest("nvarchar(max)");
 
 echo "Test successful.\n";
 
-function PopulateTestTable($conn, $params) {
+function populateTestTable($conn, $params)
+{
+    $tblName = $params['tableName'];
+    $colName = $params['columnName']; 
+    $fieldType = $params['fieldType'];
     
-    DropTestTable($conn, $params);
-    CreateTestTable($conn, $params);
+    // Create a test table of a column of type 'nvarchar(MAX)'
+    $columns = array(new AE\ColumnMeta('nvarchar(MAX)', $colName));
```

Comment:
should use `$fieldType`

Reply:
No, I had misunderstood this test too. If you read the original test, it has to create a column of nvarchar(MAX) first. That's why I added a comment to explain this.

Reply:
I see it now. But if that's the case I think the original test is flawed. `$fieldType` isn't used anywhere in the test, and line 48:
```
runTest("varchar(max)");
runTest("varbinary(max)");
runTest("nvarchar(max)");
```
are actually only testing `runTest("nvarchar(max)");` 3 times

Reply:
Yes... that was why I had initially misunderstood the test too. In the original test, $field_type is never really used.

Reply:
We should fix the test right? It shouldn't take that long (maybe)